app/controllers/users_controller.rb

```diff
@@ -8,7 +8,7 @@ class UsersController < AuthenticatedController
   def confirm_account
     render status: 400, nothing: true and return unless valid_confirm_account_params?
     if user = User.find_by_confirmation_token(params[:confirmation_token])
-      user.update(name: params[:name], password: params[:password], subscribed_to_daily_digest: true, subscribed_to_personal_activity: true)
+      user.update(name: params[:name], password: params[:password])
```

Comment:
You could do something like

    user.update(params.slice(:name, :password)

or do something like

    user.update(user_params)
    ....
    private
    def user_params
      params.permit(:name, :password)
    end